src/GovUk.Education.ExploreEducationStatistics.Public.Data.Scripts/Commands/SeedDataCommand.cs

```diff
@@ -58,7 +58,8 @@ public async ValueTask ExecuteAsync(IConsole console)
             DataSetSeed.SpcEthnicityLanguage,
             DataSetSeed.SpcYearGroupGender,
             DataSetSeed.Nat01,
-            DataSetSeed.Qua01,
+            // QUA01 has filter option labels that are too long
```

Comment:
Is it worth just getting rid of this if we can't use it?

I'm sure you've discussed with Cam, but if their filter option labels are too long in this example file, doesn't this suggest that teams out there will be requiring labels of a greater length than we support?  If so, I guess that there will need to be communications about this sent out to the teams.

Reply:
I actually kept this in because I was planning on doing some performance testing with it later on, which would modifying the script a little to trim down the labels. I can do this regardless so I've just removed it as you've suggested

I did indeed discuss the length constraints with Cam and the examples of really long filter option labels were qualification titles like:

- Industry placement (315+ hours, aged 16 – 18, level 2 or 3 with a vocational or technical core aim, occupationally specific, must be studying towards a type of qualification that may align to T levels in future and includes an external wo...
- Understand the range of service provision and roles within health and social care adults and children and young people, early years and childcare
- Non regulated Adult skills formula funded provision, Entry Level, Languages, Literature and Culture of the British Isles, 7 to 12 hrs, PW A
- Non regulated Adult skills formula funded provision, Entry Level, Languages, Literature and Culture of the British Isles, Up to 2 hrs, PW A
- Non regulated Adult skills formula funded provision, Entry Level, Nursing and Subjects and Vocations Allied to Medicine, 7 to 12 hrs, PW 

These are unnecessarily long and seemed to be the FE team labelling things badly or trying to skirt around best practice. For example, there are a bunch of 'Non regulated Adult skills formula funded provision...' options that clearly contain filters that they've  not added to the data set as columns.

I think we agreed that the problem lies with FE and that the onus should be on them to sort out, rather than let these bad labels make it into the public API. I think we'll probably try and communicate to the various stats team that public API data is being held to a higher standard and should conform to best practice to make it onto the API.

We've already agreed that a max label length of 120 chars should be more than sufficient if things are being labelled correctly. However, we're open to increasing this in the future if necessary. It's easier to increase the max than it is to lower it later on!

I think we also discussed that Cam (or whoever) would implement the various max length constraints in the screener as well to stop excessively long stuff making it into the platform at all.

Reply:
Brill sounds like a good plan